src/errors/UnexpectedError.vue

```diff
@@ -71,7 +64,7 @@ const route = useRoute()
 
   &__button {
     width: 215px;
-    padding: var(--space-3) var(--space-5);
+    color: #fff !important;
```

Comment:
Is the important needed? Any other way to achieve it?

Reply:
Yes, it's needed. It was like this before and I left it unchanged. I believe there is a separate task to handle these issues #7